<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat TCP/IP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowserChat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>771</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendMessage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionLanzarServidor"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLanzarServidor">
   <property name="text">
    <string>Lanzar servidor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditMessage</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButtonSendMessage</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
